index.html

```diff
@@ -24,7 +24,7 @@ <h2 class="title-light">Newsletter</h2>
           </div>
           <form class="col-md-6">
             <div class="input-wrap">
-              <input class="input-main input-main--email" type="email" name="email" placeholder="Your email address"
+              <input class="input-main input-main--email" type="email" name="email" placeholder="your email address"
```

Comment:
Why do need this changes? According to design it should be "Your" not "your"

Reply:
Looks good. You can merge it as soon as you're ready.

Reply:
for pull request only